manifests/init.pp

```diff
@@ -69,6 +70,7 @@
   Array[Enum['ip','ip6']] $ip_versions = ['ip','ip6'],
   Hash[String[1],Array[String[1]]] $preserve_chains_in_tables = {},
   Enum['package','vcsrepo'] $install_method = 'package',
+  Enum['installed', 'latest', 'absent'] $package_ensure = 'latest',
```

Comment:
In general I think `latest` is a bad pattern. At least with yum it puts a lot of load on the server. We've seen large Satellite installations require a *lot* of resources because of it.

Also, you don't allow any specific version.
```suggestion
  String[1] $package_ensure = 'installed',
```

Reply:
I agree, personally I highly prefer installed as well. I thought if I change it it might be backwards incompatible, but I'm not sure? I'm happy to merge your suggestion if you think we can do it in a minor release.

Reply:
Hmm, good point. I'm fine either way.

Reply:
I think it's sane to merge since it doesn't break/change existing setups